doc/ref_model/chapters/chapter02.md

```diff
@@ -180,7 +180,7 @@ By trying to sort workloads into different categories based on the requirements
 Based on the above analysis, following cloud infrastructure profiles are proposed (also shown in **Figure 2-1** below)
 - **Basic**: For Workloads that can tolerate resource over-subscription and variable latency.
 - **Network Intensive**: For Workloads that require predictable computing performance, high network throughput and low network latency.
-- **Edge** : For Edge use cases that have special requirements, in terms of offloading some functions , very low network latency ,very high storage requirements and very high computing requirements "GPU" .
+- **Edge** : For Edge use cases that have special requirements, in terms of offloading some functions, very low network latency, very high storage requirements and very high computing requirements "GPU".
```

Comment:
Does Edge have "very high storage requirements"? Many Edge use cases do not have even medium storage requirements. 
I think "GPU" needs to be deleted.

```suggestion
- **Edge** : For Edge use cases that have special requirements, in terms of offloading some functions, very low network latency and very high computing performance requirements.
```

Reply:
@pgoyal01 for vRAN use cases there is no need for GPU , but for the edge to make it more wider i belive GPU required for other use cases 

i will copy karim comments 

karim quote
"
Hello Gergely, we have the use case of the remote Edge Survellience with video analytics. Where the live streams need to be analyzed at the Edge (ML-assisted) and alarms are raised for the suspected scenes. It is one of the 5G use cases for Mission Critical IoT."

what do u think

Reply:
@ASawwaf we need to reword the sentence. There is also the issue about the " very high storage requirements" for the Edge. 

```suggestion
- **Edge** : For use cases that have special requirements, in terms of offloading some functions, very low network latency, and very high computing performance requirements. For example, the use of GPU for remote surveillance with video analytics where the live streams are, ML-assisted, analyzed at the Edge and alarms are raised for the suspected scenes. 
```

Reply:
GPUs should be in a different workload profile. One could request that hostgroups supporting the GPU workload profile be deployed in edge datacenters.

Reply:
Similar for storage requirements.  If high storage requirements are needed, they should be identified as a separate workload profile, with a supporting host aggregate, and then that can be deployed at the edge if needed.